packages/angular_devkit/core/src/_golden-api.ts

```diff
@@ -51,3 +51,7 @@ export * from './utils/lang';
 export * from './virtual-fs/path';
 export * from './virtual-fs/host/index';
 // End virtualFs namespace
+
+// Start workspace namespace
+export * from './workspace';
```

Comment:
This should be `workspace/index` are api guardian uses classic module resolution

Reply:
It passes though?

Reply:
Yeah, cause API guardian completely ignores such exports/imports.

Reply:
Updated.